README.md

```diff
@@ -16,7 +16,7 @@ To include videojs-flashls-source-handler on your website or web application, us
 
 This plugin requires the use of a [customized SWF](https://github.com/brightcove/videojs-flashls-swf) that is not included in videojs-flash.
 
-The SWF is available on the unpkg CDN https://unpkg.com/@brightcove/videojs-flashls-swf/dist/video-js.swf and is provided in this project's `dist/` folder. We highly recommend hosting the SWF yourself as unpkg does not provide uptime or support guarantees.
+The SWF is available on the unpkg CDN https://unpkg.com/@brightcove/videojs-flashls-source-handler/dist/video-js.swf and is provided in this project's `dist/` folder. We highly recommend hosting the SWF yourself as unpkg does not provide uptime or support guarantees.
```

Comment:
I'm not sure if this URL is correct.  We could get the namespace change out, publish it, and use the versioned url here.